qemu/tests/qsd_block_mirror.py

```diff
@@ -0,0 +1,100 @@
+import logging
+
+from avocado.utils import memory
+
+from provider import backup_utils
+from provider.blockdev_mirror_wait import BlockdevMirrorWaitTest
+from provider.qsd import QsdDaemonDev
+
+LOG_JOB = logging.getLogger('avocado.test')
```

Comment:
Do we need it here? No use it anywhere else.

Reply:
Remove the logging related, thanks @XueqiangWei